sympy/printing/mathml.py

```diff
@@ -466,6 +475,12 @@ class MathMLPresentationPrinter(MathMLPrinterBase):
     """
     printmethod = "_mathml_presentation"
 
+    def __init__(self, settings=None):
+        Printer.__init__(self, settings)
```

Comment:
Here, it is better to call MathMLPrinterBase instead. However, this is just for future reference as this is already done in #15906. (Even better is `super(MathMLPresentationPrinter, self).__init__(self, settings)`.)

Reply:
okay